docs/README.md

```diff
@@ -1,4 +1,5 @@
-# Bash-it
+![docker](https://github.com/Bash-it/media/raw/master/media/Bash-it.png)
```

Comment:
You've made the alt text `docker`, and I think you actually want `logo`.

Reply:
whoops, nice catch!